backend/src/packages/users/user.repository.ts

```diff
@@ -163,12 +164,10 @@ class UserRepository implements IRepository {
   }
 
   public async getAllAuthors(): Promise<UserDetailsModel[] | null> {
-    const authors = await UserDetailsModel.query().distinctOn('userId')
-      .whereRaw(`
-      EXISTS(SELECT 1
-      FROM articles,user_details
-      WHERE articles.user_id = user_details.user_id)
-    `);
+    const authors = await UserDetailsModel.query()
+      .from(getAuthorsSubquery())
```

Comment:
Let's keep this query in repository - as from helper it is used only single time here